package.json

```diff
@@ -29,58 +21,42 @@
     }
   },
   "engines": {
-    "npm": "please_use_yarn_instead"
+    "npm": "please_use_pnpm_instead"
```

Comment:
this should be removed, and `pnpm` engine should be set

Reply:
https://nodejs.org/api/packages.html#packagemanager